ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+import java.util.Random;
+
+public enum Color {
+    RED, BLUE, GREEN, YELLOW, ORANGE;

```

Comment:
color enum should be in a separate file, this class should be ColorSupplier class